src/system.rs

```diff
@@ -39,6 +40,60 @@ pub enum SystemdSystemState {
     offline = 7,
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Default, Eq, PartialEq)]
+pub struct SystemdVersion {
+    major: u32,
+    minor: String,
+    revision: Option<u32>,
+    os: String,
+}
```

Comment:
@DaanDeMeyer - Is there any official docs about the versioning? My googling let me down. Happy to correct naming and typing here if I've guessed wrong.